test/dynamo/test_functions.py

```diff
@@ -1839,13 +1840,13 @@ def call_triton_return_view(x: torch.Tensor):
         t_view = t.view(16)
 
         compiled_func = torch.compile(
```

Comment:
Should add some more concrete tests about recompilation here with dynamic shapes.

Reply:
will do as follow up